pybaseball/lahman.py

```diff
@@ -1,9 +1,12 @@
-import requests	
-from zipfile import ZipFile	
-import os
-import pandas as pd
 from io import BytesIO
+from os import path
 from typing import Optional
+from zipfile import ZipFile
+
+import pandas as pd
+import requests
+
+from .datahelpers import caching
```

Comment:
I think Lahman should be treated separately, as a properly stable database.

Reply:
I removed the one cache wrapper in here, but think keeping the use of the caching directory here makes sense.

Reply:
I think we should use the `download_lahman`, but point to `.pybaseball/` so it ends up like `.pybaseball/baseball-databank` or whatever it is